content/en/docs/setup/getting-started/index.md

```diff
@@ -182,10 +184,19 @@ The following tasks are a good place for beginners to start:
 - [Accessing external services](/docs/tasks/traffic-management/egress/egress-control/)
 - [Visualizing your mesh](/docs/tasks/observability/kiali/)
 
-The next step is to customize Istio and deploy your own applications.
-Before you install and customize Istio to fit your platform and intended use, check out
-our [deployment models](/docs/setup/deployment-models/) and [general installation](/docs/setup/)
-documents.
+The tasks above provide a good basic understanding of Istio's features but,
+explore more [tasks](/docs/tasks/) depending on what interests you.
```

Comment:
I intentionally removed this in a previous PR because it is repeating what was said just above, but I guess we can merge it as is and fix this wording in a followup PR.